sql/tests: TestRandomSyntaxFunctions failed

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/20259364?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/20259364?buildTab=artifacts#/) on master @ [3568bd584e6bb917debe0faf0e98bc1686750f51](https://github.com/cockroachdb/cockroach/commits/3568bd584e6bb917debe0faf0e98bc1686750f51):


```
*   | 	src/runtime/asm_amd64.s:1700
* Wraps: (5) invalid type family:TriggerFamily width:0 precision:0 locale:"" visible_type:0 oid:2279 time_precision_is_set:false 
* Error types: (1) *withstack.withStack (2) *withstack.withStack (3) *assert.withAssertionFailure (4) *withstack.withStack (5) *errutil.leafError
*
    stopper.go:226: -- test log scope end --

ERROR: a panic has occurred!
Details cannot be printed yet because we are still unwinding.
Hopefully the test harness prints the panic below, otherwise check the test logs.

test logs left over in: /artifacts/tmp/_tmp/d437d2c847dfedbc4972f231c3331c8e/logTestRandomSyntaxFunctions300555912
    stopper.go:226: panic: invalid type family:TriggerFamily width:0 precision:0 locale:"" visible_type:0 oid:2279 time_precision_is_set:false 
--- FAIL: TestRandomSyntaxFunctions (1.19s)
panic: invalid type family:TriggerFamily width:0 precision:0 locale:"" visible_type:0 oid:2279 time_precision_is_set:false  [recovered]
	panic: invalid type family:TriggerFamily width:0 precision:0 locale:"" visible_type:0 oid:2279 time_precision_is_set:false  [recovered]
	panic: invalid type family:TriggerFamily width:0 precision:0 locale:"" visible_type:0 oid:2279 time_precision_is_set:false  [recovered]
	panic: invalid type family:TriggerFamily width:0 precision:0 locale:"" visible_type:0 oid:2279 time_precision_is_set:false  [recovered]
	panic: invalid type family:TriggerFamily width:0 precision:0 locale:"" visible_type:0 oid:2279 time_precision_is_set:false 

goroutine 4528082 [running]:
testing.tRunner.func1.2({0x7038ea0, 0xc02f59d278})
	GOROOT/src/testing/testing.go:1632 +0x230
testing.tRunner.func1()
	GOROOT/src/testing/testing.go:1635 +0x35e
panic({0x7038ea0?, 0xc02f59d278?})
	GOROOT/src/runtime/panic.go:791 +0x132
github.com/cockroachdb/cockroach/pkg/util/leaktest.AfterTest.func2()
	pkg/util/leaktest/leaktest.go:136 +0x3aa
panic({0x7038ea0?, 0xc02f59d278?})
	GOROOT/src/runtime/panic.go:791 +0x132
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).recover(0x451cc5?, {0x85c6088, 0xd1a36a0})
	pkg/util/stop/stopper.go:226 +0x65
panic({0x7038ea0?, 0xc02f59d278?})
	GOROOT/src/runtime/panic.go:791 +0x132
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).Stop(0xc021edefc0, {0x85c6088, 0xd1a36a0})
	pkg/util/stop/stopper.go:566 +0x165
panic({0x7038ea0?, 0xc02f59d278?})
	GOROOT/src/runtime/panic.go:791 +0x132
github.com/cockroachdb/cockroach/pkg/util/ctxgroup.Group.Wait({0xc0267d2fc0?, {0x85c6258?, 0xc0253758b0?}, 0xc00c7303f0?})
	pkg/util/ctxgroup/ctxgroup.go:150 +0x97
github.com/cockroachdb/cockroach/pkg/util/ctxgroup.GroupWorkers({0x85c6300?, 0xc005715f10?}, 0x8, 0xc025d1cba0)
	pkg/util/ctxgroup/ctxgroup.go:210 +0x14a
github.com/cockroachdb/cockroach/pkg/sql/tests_test.testRandomSyntax(0xc035d83520, 0x0, {0x76519d3?, 0xf?}, 0x0, 0xc023752e70)
	pkg/sql/tests/rsg_test.go:961 +0x6fc
github.com/cockroachdb/cockroach/pkg/sql/tests_test.TestRandomSyntaxFunctions(0xc035d83520)
	pkg/sql/tests/rsg_test.go:447 +0x1a5
testing.tRunner(0xc035d83520, 0x7ab82a0)
	GOROOT/src/testing/testing.go:1690 +0xf4
created by testing.(*T).Run in goroutine 1
	GOROOT/src/testing/testing.go:1743 +0x390
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/sql-foundations
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

--- a/pkg/sql/tests/rsg_test.go
+++ b/pkg/sql/tests/rsg_test.go
@@ -421,6 +421,10 @@ func TestRandomSyntaxFunctions(t *testing.T) {
 					// Some spatial function are slow and testing them here
 					// is not worth it.
 					continue
+				case "trigger_in", "trigger_out":
+					// Skip trigger I/O functions since we can't generate random trigger
+					// arguments.
+					continue
 				case "crdb_internal.reset_sql_stats",
 					"crdb_internal.check_consistency",
 					"crdb_internal.request_statement_bundle",

